channels/us.m3u

```diff
@@ -1,7 +1,7 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="KOTR-LD.us" tvg-name="11 MyTV (KOTR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/b/bd/Kotrlp.jpg" group-title="Local",11 MyTV (KOTR-LD) (480i)
 https://cdnapisec.kaltura.com/p/2630451/sp/2630451/playManifest/entryId/1_hq4vlk40/deliveryProfileId/672/protocol/https/format/applehttp/a.m3u8
-#EXTINF:-1 tvg-id="13thStreet.us" tvg-name="13th Street (Poland)" tvg-country="PL" tvg-language="Polish;English" tvg-logo="" group-title="",13th Street (Poland)
+#EXTINF:-1 tvg-id="13thStreet.us" tvg-name="13th Street (Poland)" tvg-country="PL" tvg-language="Polish" tvg-logo="" group-title="Entertainment",13th Street (Poland)
```

Comment:
Not English ? There's both languages available on this one.

Reply:
technically yes. english is in the background but the overdub of polish you can listen.

Reply:
tested again. Not all shows are overdubbed. I will revert back

Reply:
If you're using VLC, you got the choice of using both undubbed / dubbed versions.

Reply:
i normally test vlc , kodi, and streamtest.in. i have reverted this change and updated pr